sale_product_deposit/data/consignes_group_tax.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="consignes_group_tax" model="account.tax.group">
```

Comment:
i would take this opportunity to rename the xmlid to english. i suggest `deposit_tax_group`. the xml file should be named `account_tax_group_data.xml`.

Reply:
done